src/components/molecules/Card/component.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from '@emotion/styled';
+import { Props } from './props';
+
+export const Card = styled.div<Props>`
+   ${({ background, color, disableBorder}) => `
+      background-color: ${ '#fff' ||  background };
+      color: ${ '#282828' ||  color };
```

Comment:
string is not false or null, so always will be set || value. 
```suggestion
      background-color: ${ background || '#fff' };
      color: ${ color || '#282828 };
```